client_it_test.go

```diff
@@ -321,3 +321,112 @@ func TestClusterReconnection_ReconnectModeOff(t *testing.T) {
 	time.Sleep(100 * time.Millisecond)
 	assert.Equal(t, false, c.Running())
 }
+
+func TestClient_GetDistributedObjects(t *testing.T) {
+	const (
+		testMapName = "get-distributed-objects-map"
+		testSetName = "get-distributed-objects-set"
```

Comment:
How about using auto-generated map and set names, like: https://github.com/hazelcast/hazelcast-go-client/blob/7326ba425837f2cefb9c16f4eb2479bc282ec301/internal/it/map.go#L35

Reply:
Do you mean using `Map/SetTester` methods or only generating this kind of names? If it's the latter, is there a particular reason to keep DDS names for tests in this format? I'm asking since I couldn't get the importance of naming. Bcz I'd say any string would work here.

Reply:
We add a random suffix to DSS names in tests, in order to not accidentally use the same name accross tests, like both `TestClient_GetDistributedObjects` and `TestClient_PurgeDestroyedDistributedObjects` use `get-distributed-objects-map` as the map name.

Reply:
Doesn't using `it.StartNewClusterWithOptions` in both tests ensure isolation? Otherwise these newly added tests would not work at all - as it asserts the number of objects created within the test body. It of course makes sense to use different names for the tests using the same cluster, but I was asking for the two tests I just added.